index.css

```diff
@@ -69,3 +69,216 @@
   font-size: 16px;
   line-height: 24px;
 }
+
+.achivement {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.recent {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 40;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+.works {
+  display: grid;
+}
```

Comment:
- [ ] Please know that you should be using the margins specified in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2154), so please do that. 😁😁